packages/docusaurus-theme-classic/src/theme/NavbarItem/DefaultNavbarItem.tsx

```diff
@@ -72,9 +72,11 @@ export function NavLink({
 function DefaultNavbarItemDesktop({
   className,
   isDropdownItem = false,
+  type: _type,
+  position: _position,
```

Comment:
```
<a class="navbar__item navbar__link" type="doc" position="left" href="/docs/cli">
```

In addition, extra attributes are added to rendered regular link: `type` and `position`. Is there any other (better) way to remove them?

Reply:
The extra attributes exist because they are not destructed in `NavbarItem/index.tsx`... Should be removed there

Reply:
That's better, thanks.